conda_forge_tick/models/pr_json.py

```diff
@@ -48,7 +52,7 @@ class PullRequestState(StrEnum):
     """
 
 
-class PullRequestInfoHead(StrictBaseModel):
+class PullRequestInfoHead(ValidatedBaseModel):
```

Comment:
Why did you relax this?

Reply:
In `GitHubBackend.create_pull_request`, the pr_json pydantic model is directly validated on the JSON returned by the GitHub API. However, we do not want to keep all the information the API returns, mainly since these pydantic models also correspond to the data stored in cf-graph. Hence, I relaxed this to ignore any additional data returned by GitHub.
Since the direct usage of the model also prevents the code from introducing additional extra fields to the pr_json data, it's OK that the model validation test can no longer detect such extra fields when processing the cf-graph data.